README.md

```diff
@@ -17,7 +17,8 @@ To build it, we need extra dependencies namely bcc-devel and kernel-headers for
 Interface:
 
 ```
-sudo podman run --annotation io.containers.trace-syscall=[absolute path to the json file] IMAGE COMMAND
+sudo podman run --annotation io.containers.trace-syscall="if:[absolute path to the input file];of:[absolute path to the output file]" IMAGE COMMAND
 ```
 
-The profile will be created at the path provided to the annotation.
+The tool can also extend extending SECCOMP profiles by reading the input file, and extend it by adding the recorded syscalls to it.
```

Comment:
typo: " extend extending"

This is a bit hard to read. I think it'll be easier to understand by stating that an input file can be used to create a baseline and that newly recorded syscalls will be added to this set (and written to the output). Also, it's important to note whether it will cause errors in case the input file explicitly disables a syscalls that is later being recorded while tracing.

Reply:
> This is a bit hard to read.

Fixing :D

> Also, it's important to note whether it will cause errors in case the input file explicitly disables a syscalls that is later being recorded while tracing.

Nice catch! I didn't think about this! I will make this more explicit in the documentation.